Sources/SwiftProtobuf/Google_Protobuf_Any+Registry.swift

```diff
@@ -105,7 +107,7 @@ extension Google_Protobuf_Any {
     @discardableResult public static func register(messageType: Message.Type) -> Bool {
         let messageTypeName = messageType.protoMessageName
         var result: Bool = false
-        knownTypesQueue.sync(flags: .barrier) {
+        let block = {
```

Comment:
I wonder if there's some way to use a file local helper with all the conditionals that takes a closure to keep the main part of the code simple and hide the details.